test/RelayServer.webpack.test.ts

```diff
@@ -7,8 +7,11 @@ describe('RelayServer-webpack', () => {
   before('create webpack', function () {
     this.timeout(15000)
     const jsrelayDir = path.join(__dirname, '..', 'jsrelay')
+    console.log('jsrelayDir=', jsrelayDir)
+    // const webpackEntry = require( jsrelayDir+'/webpack.config').entry
+    // assert.ok( fs.existsSync(jsrelayDir+'/'+webpackEntry), 'missing entry file: '+webpackEntry)
     fs.rmdirSync(path.join(jsrelayDir, 'dist'), { recursive: true })
-    childProcess.execSync('npx webpack', { cwd: jsrelayDir })
+    childProcess.execSync('sh -c "find . ../dist ; npx webpack"', { cwd: jsrelayDir, stdio: 'inherit' })
```

Comment:
Why? it does nothing other than print the current folder and ../dist

Reply:
it was added to troubleshoot a bug in the webpack. (it passed locally, and failed on the server. I don't really remember what fixed it..)

Reply:
I suggest removing things you cannot remember being useful.